packages/web/components/Technology/Details/Tables/Costs.js

```diff
@@ -42,7 +42,7 @@ const Costs = ({ title, data, totalColor, containerHeight }) => {
 		type: getTypeLabelByValue(item?.type),
 		quantity: item?.quantity,
 		value: item?.value,
-		total: item?.quantity * item?.value,
+		total: formatCurrencyToInt(item?.value || 0) * parseInt(item?.quantity || 0, 10),
```

Comment:
I don't know if we really need this fallback for quantity value. I suppose that will be more useful prevent errors declaring the default quantity as one. What did you think about it?
```suggestion
		total: formatCurrencyToInt(item?.value || 0) * parseInt(item?.quantity || 1),
```

Reply:
We'll stay with this fallback as discussed in Slack 🤝